docs/reference/errors-and-warnings/NU5130.md

```diff
@@ -0,0 +1,47 @@
+---
+title: NuGet Warning NU5130
+description: NU5130 Warning code
+author: zivkan
+ms.date: 09/16/2019
+ms.topic: reference
+ms.reviewer: karann
+f1_keywords:
+  - NU5130
+---
+
+# NuGet Warning NU5130
+
+<pre>WARNING: NU5130: Some target frameworks declared in the dependencies group of the nuspec and the lib/ref folder have compatible matches, but not exact matches in the other location. Unless intentional, consult the list of actions below:
```

Comment:
Do we add the text `WARNING: NU5130: ` in the docs page?

Reply:
no we don't. Good catch!